wgpu-bindings/bindings.h

```diff
@@ -0,0 +1,112 @@
+#include <cstdint>
+#include <cstdlib>
+
+enum class PowerPreference {
+  Default = 0,
+  LowPower = 1,
+  HighPerformance = 2,
+};
+
+template<typename B>
+struct Buffer;
+
+template<typename B>
+struct CommandBuffer;
+
+template<typename B>
+struct ComputePass;
+
+template<typename B>
+struct Device;
+
+template<typename B>
+struct RenderPass;
+
+template<typename B>
+struct ShaderModule;
+
+struct ShaderModuleDescriptor;
+
+template<typename T>
+struct Handle {
+  T *_0;
+};
+
+using DeviceHandle = Handle<Device<B>>;
+
+using AdapterHandle = Handle<Adapter<B>>;
+
+struct Extensions {
+  bool anisotropic_filtering;
+};
+
+struct DeviceDescriptor {
+  Extensions extensions;
+};
+
+using ComputePassHandle = Handle<ComputePass<B>>;
+
+using RenderPassHandle = Handle<RenderPass<B>>;
+
+using CommandBufferHandle = Handle<CommandBuffer<B>>;
+
+using InstanceHandle = Handle<Instance>;
+
+using BufferHandle = Handle<Buffer<B>>;
+
+struct BufferDescriptor {
+  uint32_t size;
+  BufferUsageFlags usage;
```

Comment:
bitflags aren't supported yet, see https://github.com/eqrion/cbindgen/issues/100

We could just expose `uint32_t` everywhere for flags, and map back on the Rust side...